nemo/collections/cv/modules/conv2d.py

```diff
@@ -0,0 +1,58 @@
+from typing import Union
+
+from torch import nn
+from torch.nn import init
+from torch.nn.common_types import _size_2_t
+
+
+class Conv2d(nn.Conv2d):
```

Comment:
Hmm does this really need to be a module? Its basically a copy of the PT Conv2d with just `weight_init` added

Reply:
In any case, if its added, it should go to cv/parts/ not be in cv/modules

Reply:
done, moved to cv/parts/submodules/